reana_job_controller/job_monitor.py

```diff
@@ -276,7 +276,7 @@ def watch_jobs(self, job_db, app=None):
         :param job_db: Dictionary which contains all current jobs.
         """
         while True:
-            logging.debug("Starting a new stream request to watch Jobs")
+            logging.info("Starting a new stream request to watch Jobs")
```

Comment:
Is the change from "debug" to "info" intended on purpose?

Reply:
Yes, in this way we can check that when an error occurs the job monitor correctly restarts. Otherwise we would only see the error message, without knowing whether another "watch" request has been made to the Kubernetes API